### ISSUE TITLE
update dc/os catalog to latest cockroachdb

### ISSUE THREAD

**Issue by kannanlakshmi** [cockroachdb/dcos-cockroachdb-service#74]
Mesosphere has requested we update the DC/OS universe to the latest CockroachDB as they have an active customer POC utilizing CRDB through the universe. They are currently on 2.0.1 Alex sent instructions over email on how to do this to Bob, myself and Ben.

**Comment by habibrosyad** [comment]
How is the progress with this one?

**Comment by kannanlakshmi** [comment]
Hi @habibrosyad this is not a priority for us at the moment, the issue is mostly for tracking purposes.

**Comment by a-nldisr** [comment]
@kannanlakshmi Is the deployment method for cockroachdb still similar as when it was v2.0.1. Im evaluating the DCOS package, if it would be possible to just replace the binaries with the latest version then it could help the DCOS community to keep it up to date easy.

**Comment by kannanlakshmi** [comment]
Yes, the deployment method is still similar. We have currently de-prioritized the DC/OS package, so unfortunately this is not a top priority item on our roadmap.
